simplemonitor/util/__init__.py

```diff
@@ -236,15 +236,19 @@ def all_types() -> list:
     return (register, get_class, all_types)
 
 
-def check_group_match(group: str, group_list: List[str]) -> bool:
+def check_group_match(group: Union[str, List[str]], group_list: List[str]) -> bool:
     """
     Check if a group is contained in the group list.
 
     If the group list is a single element, "_all", then it matches.
     """
     if group_list[0] == "_all":
         return True
-    if group in group_list:
+
+    if isinstance(group, str):
```

Comment:
Fancy adding a test or two around this? I'm happy there's no regressions since it seems the original tests pass but good to check this works as intended.

Reply:
Sure thing.